inlet/core/enricher.go

```diff
@@ -207,7 +226,11 @@ func (c *Component) writeExporter(flow *schema.FlowMessage, classification expor
 	return true
 }
 
-func (c *Component) classifyExporter(t time.Time, ip string, name string, flow *schema.FlowMessage) bool {
+func (c *Component) classifyExporter(t time.Time, ip string, name string, flow *schema.FlowMessage, classification exporterClassification) bool {
+	// we already have the info provided by the metadata component
+	if classification.Group != "" || classification.Role != "" || classification.Site != "" || classification.Region != "" || classification.Tenant != "" {
```

Comment:
This could be `classification != exporterClassification{}` (I think)

Reply:
I guess (not sure) this is an extra memory allocation on the hot path ?

Reply:
Being a const, I suppose the compiler would do a good job doing the same thing as the expanded version. It's just the day you add something to this struct, it's one less site to modify.

Reply:
done